src/main/java/edu/ucsb/cs156/happiercows/controllers/CommonsController.java

```diff
@@ -48,7 +48,7 @@
   ObjectMapper mapper;
 
   @ApiOperation(value = "Get a list of all commons")
-  @PreAuthorize("hasRole('ROLE_USER')")
+  //@PreAuthorize("hasRole('ROLE_USER')")
```

Comment:
Maybe consider removing this commented out code?  

I can understand why you left it in until now, but I think now it's time to commit to this direction.

Reply:
This well leak users, we need a dedicated public method in the controller.

Reply:
It's not a neat solution, but I think any team can double back in the near future to resolve this if it causes problems. We're also assuming that this game isn't necessarily 'public' anyway, as only students in the same class should be able to use it.